OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -158,6 +158,20 @@ public func isMethodDELETE() -> OHHTTPStubsTestBlock {
 }
 
 /**
+ * Matcher for testing an `NSURLRequest`'s **absolute url**.
+ *
+ * e.g. the absolute url is `https://api.example.com/signin` in `https://api.example.com/signin`
+ *
+ * - Parameter scheme: The absolute url to match
+ *
+ * - Returns: a matcher (OHHTTPStubsTestBlock) that succeeds only if the request
+ *            has the given absolute url
+ */
+public func isAbsoluteUrl(_ url: String) -> OHHTTPStubsTestBlock {
```

Comment:
I prefer `isAbsoluteURLString` over `isAbsoluteString`, it doesn't sound too bad :)